packages/gatsby/src/commands/develop.ts

```diff
@@ -295,7 +295,7 @@ module.exports = async (program: IProgram): Promise<void> => {
     null
   )
 
-  let unlocks: Array<UnlockFn> = []
+  let unlocks: Array<UnlockFn | null> = []
```

Comment:
While this might fix typing, it doesn't seem like we actually handle this type correctly in https://github.com/gatsbyjs/gatsby/blob/add4e344482411ec5f63b14431c1558aa0a3d22f/packages/gatsby/src/commands/develop.ts#L501-L527 (due to first param being untyped?) - i.e. we do call `unlock()` on each item, even tho apparently it might be `null`?

Reply:
Ok, will add the typing and see what it complains about. Probably that we should handle `null`

Reply:
Added!

Reply:
Resolved with https://github.com/gatsbyjs/gatsby/pull/31519/commits/5f9743e2124873a82504ec6e270693c2c6a550f5